src/markdown_generators.js

```diff
@@ -323,7 +326,10 @@ const generateResourceMarkdownForVideo = (media, courseData, pathLookup) => {
       video_captions_file:   captionsFileLocation,
       video_transcript_file: transcriptFileLocation,
       archive_url:           archiveUrl
-    }
+    },
+    optional_text:       optionalText,
+    optional_tab_title:  media["optional_tab_title"] || "",
+    resource_index_text: media["resource_index_text"] || ""
```

Comment:
I think `related_resources_text` needs to be copied over too? It is mentioned in https://github.com/mitodl/ocw-data-parser/issues/162, was already included in parsed json